waiter/src/waiter/scheduler/kubernetes.clj

```diff
@@ -688,8 +685,10 @@
            replicaset-api-version service-id->password-fn] :as scheduler}
    service-id
    {:strs [backend-proto cmd cpus grace-period-secs health-check-interval-secs health-check-max-consecutive-failures
-           health-check-port-index health-check-proto image mem min-instances ports run-as-user] :as service-description}
-   {:keys [container-init-commands default-container-image log-bucket-url image-aliases] :as context}]
+           health-check-port-index health-check-proto image mem min-instances namespace ports run-as-user]
+    :as service-description}
+   {:keys [container-init-commands default-container-image default-namespace log-bucket-url image-aliases]
+    :as context}]
   (let [work-path (str "/home/" run-as-user)
```

Comment:
Should we report an error if the default container image is missing?

Reply:
That would probably be nice, but in practice, our integration tests will all fail if the image isn't set, and we'll figure out what's wrong pretty quickly by the errors in the logs.

Reply:
I meant default namespace.

Reply:
Added sanity checks.